osu.Game/Localisation/CommonStrings.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class CommonStrings
+    {
+        private const string prefix = "osu.Game.Localisation.Common";
+
+        /// <summary>
+        /// "Cancel"
+        /// </summary>
+        public static LocalisableString Cancel => new TranslatableString(getKey("cancel"), "Cancel");
```

Comment:
Right now this setup does *not* required resx files existing for english. I tried to make that clear in my PR description. The idea being that it can be 100% implemented with c# code alone, and the underlying resources would be generated in the crowdin sync pass.

There are issues with resx tooling:

- It requires IDE tooling or raw text editing (as you mentioned)
- It generally created a `.Designer` file, which we don't need/want

I think personally it makes the process more convoluted during implementation, but there's still room for discussion.

Reply:
In addition, tooling for `.resx` support doesn't seem to work so well (followed up in https://youtrack.jetbrains.com/issue/RSRP-476154 for the specific case I ran into), and when it *does* work, the code which is dropped in in place will be a direct reference to the `.Designer` model, rather than our own. So I'm not sure how helpful that is, unless we have a post-process step that goes through and replaces the designer code references with the `OsuStrings` ones? Feasible, but not sure.

Reply:
One thing I'd like to add is that osu-stable stores its translations in a `en.txt` file as @bdach describes, but over the years I found this to result in a generally bad coding experience. Searching for what string some button displays, and then editing it tends to get quite tedious.

So I really agree with the way it's done here, it's just one go-to-definition keybind away, or one mouse hover over the item because of the xmldoc.

Reply:
The navigation benefits I can understand for sure. My main concern is that I'd loathe to have to add a C# entry, and then go separately add a resx entry manually. I don't much care in which direction the automatic generation goes, but IMO the two should have one editable source of truth.

Reply:
You wouldn't add a resx entry as I understand. The source of truth for the english translation would be the C#.

Conversion would be resx -> c# and c# -> resx, but only for uploading/import from/to CrowdIn.

Reply:
Ok, so am I to understand that the English resx files that are a part of this PR are or eventually will be auto-generated artifacts that will only be exchanged with crowdin and never touched during dev? If that is the case I think that completes the puzzle in my head.

Reply:
That is correct. Only included in this PR because it was throwing otherwise (although I have since resolved that, so I believe they can be removed now, although I have not tested if that inhibits the other example locales being found or not yet).